measurement/validation.py

```diff
@@ -240,7 +240,7 @@ def generate_validation_report(
     Returns:
         A tuple with the summary report and the granular report.
     """
-    period = Station.objects.get(station_code=station).delta_t.delta_t
```

Comment:
@dalonsoa I did remove all the references, but it's failing at this point 
![image](https://github.com/user-attachments/assets/6fd3c4b0-cadd-41f6-990c-ecab00f31e6e)
Because I removed the lines above

Reply:
Would we still need to keep the delta_t field in the model and make it non-editable by the user?

Reply:
No, you need to edit things further and work with the meaning of `delta_t`. So, that function is called in here: https://github.com/ImperialCollegeLondon/paricia/blob/7db743b69ae17255dc192fe617d3f0b762d5428a/measurement/reporting.py#L271

As you have deleted `delta_t`, period does not make sense, anymore, as it cannot be `Station` - it was meant to be a number¬ So you need to delete all references and functionality related to `period` as well.